### ISSUE TITLE
Replying on live chat

### ISSUE THREAD

**Issue by paolog22** [RocketChat/Rocket.Chat#8557]
### Description:
Replying to a live chat

### Server Setup Information:

Rocket.Chat Version: 0.59.1
NodeJS Version: 4.8.4 - x64
Platform: linux
ReplicaSet OpLog: Enabled
Commit Hash: 0710c94d6a
Commit Branch: HEAD 

### Steps to Reproduce:

1. Chat to live chat
2. Reply the chat from rocket chat livechat agent

### Expected behavior:

should send a reply to the live chat

### Actual behavior:

cant send chat

### Relevant logs:
`Error invoking Method 'sendMessage': Internal server error [500]`
https://rocket.chat/docs/contributing/reporting-issues#gathering-logs -->

**Comment by dnebrich** [comment]
I'm also seeing this.  Also saw it with earlier 0.59 RC releases.  My error message is slightly different, see below.  With 0.58, when clicking on a guest in the live chat pool, a notification would pop-up to accept it.  Now, when clicking on it it opens the live chat window.  When trying to reply, the text sent in reply is greyed out.  This makes live chat unusable for me.

Exception while invoking method 'sendMessage' TypeError: Cannot read property 'blocker' of undefined   at [object Object].Meteor.methods.sendMessage (/snap/rocketchat-server/1172/programs/server/packages/rocketchat_lib.js:15629:59)   at [object Object].methodsMap.(anonymous function) (/snap/rocketchat-server/1172/programs/server/packages/rocketchat_lib.js:1276:26)   at [object Object].methodMap.(anonymous function) (packages/rocketchat_monitoring.js:2731:30)   at maybeAuditArgumentChecks (/snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:1857:12)   at /snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:903:20   at [object Object].EVp.withValue (packages/meteor.js:1134:15)   at /snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:902:47   at [object Object].EVp.withValue (packages/meteor.js:1134:15)   at /snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:901:46   at [object Object]._.extend.protocol_handlers.method (/snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:874:21)   at /snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:753:85

**Comment by internalfx** [comment]
I have this as well
`Exception while invoking method 'sendMessage' TypeError: Cannot read property 'blocker' of undefined`

I also cannot hide livechat rooms, any help appreciated.

**Comment by digitalextremist** [comment]
Here as well:

```
Exception while invoking method 'sendMessage' TypeError: Cannot read property 'blocker' of undefined   at [object Object].Meteor.methods.sendMessage (/snap/rocketchat-server/1172/programs/server/packages/rocketchat_lib.js:15629:59)   at [object Object].methodsMap.(anonymous function) (/snap/rocketchat-server/1172/programs/server/packages/rocketchat_lib.js:1276:26)   at [object Object].methodMap.(anonymous function) (packages/rocketchat_monitoring.js:2731:30)   at maybeAuditArgumentChecks (/snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:1857:12)   at /snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:903:20   at [object Object].EVp.withValue (packages/meteor.js:1134:15)   at /snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:902:47   at [object Object].EVp.withValue (packages/meteor.js:1134:15)   at /snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:901:46   at [object Object]._.extend.protocol_handlers.method (/snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:874:21)   at /snap/rocketchat-server/1172/programs/server/packages/ddp-server.js:753:85  
```

**Comment by joserribeiro26** [comment]
I have same problem now

**Comment by sampaiodiego** [comment]
thanks guys for reporting it.

this was already fixed by #8577 

expect the fix being released today on `0.59.2` version
